### DISCUSSION TITLE
Could the "--cut-off" option deal with the bulk materials?

### DISCUSSION THREAD

**Discussion by alystone** [tongzhugroup/mddatasetbuilder#192] [Q&A]
In the present verison of datasetbuilder, it seems that the only a complete molecule or free radical will be taken with a cutoff value. This leads to my problem. My system invovles the heterogenerous catalysis which contains a slab solid materials. When I use datasetbuilder, even use a small cutoff value, and the generated gjf invovles more than 1500 atoms, which cannot deal with Gaussian and MN15 method. I wonder, if the mddatasetbuilder could fix that in an updated version, to provide a option to realize breaking the chemical bond in solid materials. I really appreciate it.

**Comment by njzjz** [comment]
Do you have any ideas to calculate the bulk materials using Gaussian?

**Reply by alystone** [reply]
It's a challenging task, but i think the clusterization method also works. My system invovles the materials cosistent of only 2p elements (B、C、N、O)，which may simplify things a bit. The difficuty with Gaussian seems to lie in setting the multiplicity. I think a feasible approach might be to extract clusters by breaking bonds from the cut-off, preserving dangling bonds. Based on the current multiplicity computation with mddatasetbuilder, we could add the contribution of the number of these dangling bonds, setting a high-spin state multiplicity to ensure Gaussian can normally continue calculations for the core material. 
Alternatively, have mddatasetbuilder and deepmd-kit already implemented fragmentation method for bulk materials (Not for pure bulk, but for heterogenerous surface) in VASP? This method does not require setting multiplicity. If yes, it will be very helpful for me.
